src/translations/strings.yaml

```diff
@@ -737,6 +737,9 @@ accessibility:
       - Used by screen readers to denote that a particular item is currently selected.
       - For example, given a segmented control with 2 items, where the first item is currently selected. We want the screen reader to read out "Selected, <item name>".
       - The %1 represents the name of the selected item.
+  selected:
+    value: "Selected"
```

Comment:
You don't need quotes, there are no special characters here.

Reply:
Thanks, fixed.